libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/drillDown.spec.ts

```diff
@@ -128,6 +140,95 @@ describe("Drilling", () => {
+                .should(
+                    "have.text",
+                    `table has row column measure › ` +
+                        `${assertTableValues[0]}` +
+                        " › " +
+                        `${assertDrillToProduct[0]}`,
+                );
+            assertDrillToYear.forEach((value, index) => {
+                drillModal.getTable().hasCellValue(index, 0, value);
+            });
+            drillModal.back();
+            drillModal.getTable().waitLoaded().click(1, 0);
+            drillModal
+                .getTitleElement()
+                .should(
+                    "have.text",
+                    `table has row column measure › ` +
+                        `${assertTableValues[0]}` +
+                        " › " +
+                        `${assertDrillToProduct[1]}`,
+                );
+            drillModal.close();
+            api.setUpDrillDownAttribute(PRODUCT_ID_DRILLDOWN);
+        });
+
+        it("Drilling down on a table that has two drillable attributes separate", () => {
+            const assertDrillToQuarterValue = ["Q2/2010"];
+
+            Navigation.visit("dashboard/implicit-drill");
+            api.setUpDrillDownAttribute(PRODUCT_ID_DRILLDOWN, YEAR_CLOSE);
+            firstWidget.waitTableLoaded().getTable().click(0, 0);
+            drillModal
+                .getTitleElement()
+                .should("have.text", `Table with years › ` + `${assertDrillToYear[0]}`);
+            drillModal.getTable().waitLoaded().click(0, 0);
+            drillModal
+                .getTitleElement()
+                .should(
+                    "have.text",
+                    `Table with years › ` +
+                        `${assertDrillToYear[0]}` +
+                        ` › ` +
+                        `${assertDrillToQuarterValue[0]}`,
+                );
+            drillModal.getTable().hasCellValue(0, 0, "Jun 2010");
+            drillModal.close();
+            firstWidget.waitTableLoaded().getTable().click(0, 1);
+            drillModal
+                .getTitleElement()
+                .should("have.text", `Table with years › ` + `${assertTableValues[0]}`);
+            assertDrillToProduct.forEach((value, index) => {
+                drillModal.getTable().hasCellValue(index, 0, value);
+            });
+            drillModal.getTable().waitLoaded().click(0, 0);
+            drillModal
+                .getTitleElement()
+                .should(
+                    "have.text",
+                    `Table with years › ` + `${assertTableValues[0]}` + " › " + `${assertDrillToProduct[0]}`,
+                );
+            drillModal.back();
+            assertDrillToProduct.forEach((value, index) => {
+                drillModal.getTable().hasCellValue(index, 0, value);
+            });
+            api.setUpDrillDownAttribute(PRODUCT_ID_DRILLDOWN);
+        });
+
+        it("Can not drill down with table that only has measures and columns. ", () => {
```

Comment:
remove "Can not", this test apply for drilling

Reply:
No, In case table has only measure and column, drill down can not apply.